src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -0,0 +1,116 @@
+import Boom from 'boom';
+import { get, omit } from 'lodash';
+
+import {
+  createFindQuery,
+  createFilterPath,
+  handleEsError,
+} from './lib';
+
+export class SavedObjectsClient {
+  constructor(kibanaIndex, callAdminCluster) {
+    this._kibanaIndex = kibanaIndex;
+    this._callAdminCluster = callAdminCluster;
+  }
+
+  async create(type, options = {}) {
+    const body = omit(options, 'id');
+
+    const response = await this._withKibanaIndex('index', {
+      type,
+      body
+    });
+
+    return Object.assign({
+      type: response._type,
+      id: response._id,
+      version: response._version
```

Comment:
How much thought was put into merging the type/id/version with the source of the doc when returning it? My first reaction is "elasticsearch separates them for good reason", but if this was carefully considered I'll trust that.